api/services/contact_fields_mapping_service.py

```diff
@@ -0,0 +1,119 @@
+from api import models, db, helpers
+
+
+class ContactFieldsMappingService(object):
+    def __init__(self):
+        self.user_phone = None
+
+    def set_init_data(self, jsonData):
+        user_phone = helpers.fetch_by_key("urn", jsonData["contact"])
+        self.user_phone = helpers.sanitize_phone_string(user_phone)
+
+    def handle_contact_fields_data(self, jsonData):
+        try:
+            self.set_init_data(jsonData)
+            user_details = models.User.query.get_by_phone(self.user_phone)
+            if not user_details:
+                return False
+
+            custom_fields = jsonData["contact"]["fields"]
+
+            if custom_fields:
+                for key in custom_fields:
```

Comment:
I think you can use it like this
<img width="500" alt="image" src="https://user-images.githubusercontent.com/54376313/211279698-0580d099-454e-40b4-a4e8-4a1912f63cf9.png">

```suggestion
                for field_name, field_value in custom_fields.items():
```

Reply:
Updated